elastic/apm-server
# universal profiling: reduce tombstones on stacktraces index

## Motivation/summary

In Universal Profiling we currently store a `@timestamp` field in `profiling-stacktraces` index.
This field is not used in the UI, and its presence generates a ~25% storage overhead due to tombstones when documents are replaced (host-agent periodically re-sends them).
Since we introduced ILM for K/V indices with #9558, we can remove this field.

## Checklist


For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**? N/A
- Is its use being published in **telemetry** to enable product improvement? N/A
- Have system tests been added to avoid regression? ✅ 

## How to test these changes

Steps are the same as #9558, but instead of reading metrics on ILM execution, you can should:

* add a data view for the `profiling-stacktraces-*` index pattern
* query the data view, see that no documents contain the `@timestamp` field

## Related issues
